static/js/components/SearchFacet.js

```diff
@@ -40,54 +40,58 @@ function SearchFacet(props: Props) {
         {title}
         <i className={`material-icons ${titleLineIcon}`}>{titleLineIcon}</i>
       </div>
-      {results && results.buckets && showFacetList
-        ? results.buckets.map((facet, i) => {
-          const isChecked = R.contains(facet.key, currentlySelected || [])
+      {showFacetList ? (
+        <React.Fragment>
+          {results && results.buckets
+            ? results.buckets.map((facet, i) => {
+              const isChecked = R.contains(facet.key, currentlySelected || [])
 
-          return (
+              return (
+                <div
+                  key={i}
+                  className={`${
+                    showAllFacets ||
+                      i < MAX_DISPLAY_COUNT ||
+                      results.buckets.length < FACET_COLLAPSE_THRESHOLD
+                      ? "facet-visible"
+                      : "facet-hidden"
+                  } ${isChecked ? "checked" : ""}`}
+                  onClick={() => {
+                    onUpdate({
+                      target: {
+                        name,
+                        value:   facet.key,
+                        checked: !isChecked
+                      }
+                    })
+                  }}
+                >
+                  <input
+                    type="checkbox"
+                    name={name}
+                    value={facet.key}
+                    checked={isChecked}
+                    onChange={onUpdate}
+                  />
+                  <div className="facet-label-div">
+                    <div className="facet-key">
+                      {labelFunction ? labelFunction(facet.key) : facet.key}
+                    </div>
+                    <div className="facet-count">{facet.doc_count}</div>
+                  </div>
+                </div>
+              )
+            })
+            : null}
+          {results && results.buckets.length >= FACET_COLLAPSE_THRESHOLD ? (
```

Comment:
I'm not seeing #2318 fixed here, I think you need to also check `showFacetList` here too

Reply:
are you sure? it seems to be working for me locally.

open:

![openfffasdf](https://user-images.githubusercontent.com/6207644/67427960-2de97c80-f5ab-11e9-864a-0963d04eb58f.png)

closed:

![openfffasdfasdffjfjf](https://user-images.githubusercontent.com/6207644/67427968-3346c700-f5ab-11e9-8d5d-1ed5473692ed.png)



closed:

Reply:
Ah nvm, somehow i had an older copy of the branch pulled down locally...

Reply:
haha ok I was worried there for a second :smiley: